go/rpcclientlib/obscuro_client.go

```diff
@@ -158,7 +158,7 @@ func (c *networkClient) encryptArgs(args ...interface{}) ([]byte, error) {
 func (c *networkClient) encryptParamBytes(params []byte) ([]byte, error) {
 	encryptedParams, err := ecies.Encrypt(rand.Reader, c.enclavePublicKey, params, nil, nil)
 	if err != nil {
-		return nil, fmt.Errorf("could not encrypt request params with enclave public key: %w", err)
+		return nil, fmt.Errorf("could not encrypt request params: %s, with enclave public key, error: %w", params, err)
```

Comment:
I think this might be clearer as `fmt.Errorf("could not encrypt the following request params with enclave public key: %s. Cause: %w", params, err)`